aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -80,13 +78,44 @@ def __init__(self):
         self.zones = None
         self.dtype = None
         self.names = None
-        self.name = None
+        self.name = ''
         self.description = None
         self.current_index = None
-        self.omx = False
+        self.__omx = False
         self.omx_file = None  # type: omx.File
         self.__version__ = VERSION  # Writes file version
 
+    def save(self, names=()) -> None:
+        """Saves matrix data back to file.
+
+        If working with AEM file, it flushes data to disk.  If working with OMX, requires new names
+
+        Args:
+            *names* (:obj:`tuple(str)`, `Optional`): New names for the matrices. Required if working with OMX files"""
+
+        if not self.__omx:
+            self.matrices.flush()
+            return
+
+        if isinstance(names, str):
+            names = [names]
+
+        if len(names) != len(self.view_names):
+            raise ValueError('Number of names needs to be equal to computational view')
+
+        exists = [n for n in names if n in self.names]
```

Comment:
I sometimes like any for this

```suggestion
        exists = any([n in self.names for n in names]
```

Reply:
The list is used in the message inside the if...